app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1><%= @recipe.name %></h1>
+  <p>Complexity: <%= @recipe.complexity %> </p>
+  <p>Genre: <%= @recipe.genre %> </p>
+  <h3>Ingredients List: <% @ingredients.each do |ingredient| %> </h3>
+      <p><%= ingredient.name %></p>
+    <% end %>
+
+<h3>Total Cost: <%= @total_cost %> </h3>
+
+<%= form_with url: "/recipes/#{@recipe.id}", method: :patch, local: true do |form| %>
+  <%= form.label :add_ingredient, "Add Ingredient" %>
+  <%= form.text_field :add_ingredient %>
```

Comment:
Just a suggestion, you may want to call this field what it is, instead of what it _does_; meaning `:ingredient_id` I think is clearer as to what this variable will contain in your controller.